resources/public/openapi/spec.yml

```diff
@@ -512,6 +512,9 @@ components:
           type: integer
         referee:
           type: integer
+        match-date:
```

Comment:
I guess that this match date also includes the time? It seems a bit misleading to name it match-date in the API spec.. Maybe rename to something lite match-datetime or match-time?

Reply:
Personally, I like match-date, as I think both match-time and match-datetime sounds weird. Datetime for me is just a data type, and I would not likely include that in my variable names etc.

I see your point, but I also disagree. I think I would not be surprised if I saw time to be part of a date object. Just look at [java.util.Date](https://docs.oracle.com/javase/8/docs/api/?java/util/Date.html).

However, maybe a description for match-date would be nice in the API spec.

Reply:
I agree with you on datetime, but not regarding date vs time. But it’s really not a big issue, so if I get a description then I will be fine with that!

Reply:
@nilssonemil I know this has already been merged, but I thought of a better name for the variable, that might solve both yours and my concerns: match-start? What do you think?